### PR TITLE
Grammar update

### PR DESCRIPTION
**Pull request by aminamos**
### Summary

I focused on changing instances of "setup" in comments or documentation files to assist with pull request #36222, leaving occurrences of the word in error messages or other code alone.

### CODE REVIEW

File: actionmailbox/test/dummy/bin/setup [hunk 1]
```diff
@@ -10,8 +10,8 @@ def system!(*args)
 end
 
 chdir APP_ROOT do
-  # This script is a starting point to setup your application.
-  # Add necessary setup steps to this file.
+  # This script is a starting point to set up your application.
+  # Add necessary set up steps to this file.
```
Comment (rafaelfranca) [inline actionmailbox/test/dummy/bin/setup]: I think this one is setup.

File: activerecord/examples/performance.rb [hunk 2]
```diff
@@ -70,7 +70,7 @@ def self.email
 
 # pre-compute the insert statements and fake data compilation,
 # so the benchmarks below show the actual runtime for the execute
-# method, minus the setup steps
+# method, minus the set up steps
```
Comment (rafaelfranca) [inline activerecord/examples/performance.rb]: I think here is setup

File: activerecord/lib/active_record/fixtures.rb [hunk 3]
```diff
@@ -193,7 +193,7 @@ class FixtureClassNotFound < ActiveRecord::ActiveRecordError #:nodoc:
   # When *not* to use transactional tests:
   #
   # 1. You're testing whether a transaction works correctly. Nested transactions don't commit until
-  #    all parent transactions commit, particularly, the fixtures transaction which is begun in setup
+  #    all parent transactions commit, particularly, the fixtures transaction which is begun in set up
```
Comment (rafaelfranca) [inline activerecord/lib/active_record/fixtures.rb]: I think here is setup

Comment (aminamos) [reply activerecord/lib/active_record/fixtures.rb]: Thank you!

File: actiontext/test/dummy/bin/setup [hunk 4]
```diff
@@ -10,8 +10,8 @@ def system!(*args)
 end
 
 chdir APP_ROOT do
-  # This script is a starting point to setup your application.
-  # Add necessary setup steps to this file.
```
Comment (etagwerker) [inline actiontext/test/dummy/bin/setup]: @aminamos I think this one should remain unchanged: "setup steps"

**Review by etagwerker — COMMENTED** [review]
@aminamos I think this PR is almost perfect. I only found one instance where it should be "setup"

**Comment by etagwerker** [comment]
@aminamos Hi there, are you planning to update this PR? If not, I can try submitting another one addressing the comments in here.

**Comment by aminamos** [comment]
@etagwerker feel free to take over, thank you

**Comment by etagwerker** [comment]
@aminamos Thanks! Here it is: #38441

**Pull request closed without merging**